src/views/layout.njk

```diff
@@ -3,12 +3,60 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>GOV.UK Frontend</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="/public/css/govuk-frontend.css">
   <!--[if IE 8]>
   <link rel="stylesheet" href="/css/govuk-frontend-oldie.css">
   <![endif]-->
+  <style>
+    /* Injected into the preview page, that is then put in an iframe */
+    .app-iframe-in-component-preview {
+      margin-top: 10px;
+      margin-bottom: 0;
+    }
+
+    .app-iframe-in-component-preview:before {
+      display: none;
+    }
+
+    /* TODO: Should this object have a clearfix all the time? */
+    .app-iframe-in-component-preview .govuk-o-width-container:after {
+      content: " ";
+      display: table;
+      clear: both;
+    }
+
+    .app-\!-di {
+      display: inline;
+    }
+
+    .app-c-component-preview {
+      outline: 5px solid #e4f2ff;
+    }
+
+    .app-c-component-preview__iframe {
```

Comment:
may be good to have a `min-height` here, otherwise we won't be able to preview the skip link component.

Reply:
At the moment the examples size exactly to the size they are (including margin) so you know if you've added or removed too much spacing when developing.

Could we instead a variant for skip-link that uses the `:focus` state?

Reply:
Yes, if we'll have the skip link focused we can leave it as is.